src/bin/pg_autoctl/cli_common.c

```diff
@@ -266,16 +301,29 @@ int
 keeper_cli_getopt_pgdata(int argc, char **argv)
 {
 	KeeperConfig options = { 0 };
-	int c, option_index = 0;
+	int c, option_index = 0, errors = 0;
+	int verboseCount = 0;
 
 	static struct option long_options[] = {
 		{ "pgdata", required_argument, NULL, 'D' },
-		{ "help", no_argument, NULL, 'h' },
```

Comment:
is removing --help intentional ?

Reply:
`pg_autoctl config get --help` works but `pg_autoctl config get replication.slot --pgdata primary --help` does not

Reply:
similary `pg_autoctl show state --help` but `pg_autoctl show state --formation default --help` fails with unrecognized option error.